homeassistant/components/n26/switch.py

```diff
@@ -15,12 +15,17 @@ def setup_platform(
         hass, config, add_entities, discovery_info=None):
     """Set up the N26 switch platform."""
     api_data = hass.data[DOMAIN][DATA]
```

Comment:
Please make all the same changes in this module as above.

Reply:
I've made all the changes you've mentioned, but can't pass check - it fails for Python 3.7 in module, which I've never thought to change. I suppose that failure is temporary. Can I somehow redo check later?